website/docs/components/tabs/partials/code/how-to-use.md

```diff
@@ -12,9 +12,13 @@
 </Hds::Tabs>
 ```
 
-### Custom starting tab
+### Pre-selecting a tab
 
-Customize the starting tab to display on page load. The first tab is selected by default.
+While the first tab is selected by default, it is possible to customize the starting tab to display on page load in two different ways (depending on how the tabs' state is controlled/persisted).
+
+#### Using `@isSelected` argument applied to one of the `Tab` elements
+
+By providing `@isSelected` argument to one of the `Tab` elements it's possible to declare which tab should be selected when the component is first rendered. From that moment on, the tab selection is controlled internally by the component.
```

Comment:
```suggestion
Declare which tab is selected when the component is first rendered by providing `@isSelected` argument to one of the `Tab` elements. From that moment on, the tab selection is controlled internally by the component.
```
[Suggestion]
This seems like more declarative (lack of a better term 😄 ) way to structure this.